### DISCUSSION TITLE
Relying on llvm-config

### DISCUSSION THREAD

**Discussion by micttyl** [carbon-language/carbon-lang#2162] [General]
@jonmeow
Given various ways to build and indexing, have you consider strictly specifying include headers?
I think I need to inspect more that we have some missing headers per files that are saved by either:
- llvm libraries internally include it
- llvm-config pours all of them

Have you tried manual build? How did it go?

**Comment by jonmeow** [comment]
I don't know what you mean by "manual build" (we use Bazel), or what you have in mind for "strictly specifying include headers". As far as I know, Clang doesn't support using modules.

**Reply by micttyl** [reply]
I meant, without Bazel, on a flat terminal, looking at each file and build the object by hand, combine, and so on.

**Reply by jonmeow** [reply]
Nope, not something I do. (I always use a build system)

**Reply by micttyl** [reply]
ok.
